src/components/Computer/index.js

```diff
@@ -1,8 +1,49 @@
 import React from 'react';
 import './computer.css'
+import paper from './assets/paper.png';
+import rock from './assets/rock.png';
+import scissors from './assets/scissors.png';
+import defaultImg from './assets/defaultImg.jpg';
+
+import {getRandomChoice} from '../../utils';
+
 const Computer = () =>{
+const choice = getRandomChoice();
+
+  if (choice === 'paper') {
     return (
-      <h1 className='computer'> Hello Butey Girl in Computer Part^_^ </h1>
+      <div className='computer'>
+        <h1> Computer </h1>
+        <img src={paper} alt='paper'/>
+      </div>
     )
   }
+  else if (choice === 'rock') {
+    return (
+      <div className='computer'>
+        <h1> Computer </h1>
+        <img src={rock} alt='rock'/>
+      </div>
```

Comment:
the three if(s) could be modularised if the line containing the img were written as `<img src={choice}...>`